src/js/sw/sw.js

```diff
@@ -217,3 +220,53 @@ const fetchHandler = async (event) => {
 self.addEventListener('install', precacheAssets);
 self.addEventListener('activate', clearOldCaches);
 self.addEventListener('fetch', fetchHandler);
+
+/** @type {MessagePort} */
+let messageChannelPort;
```

Comment:
Some quick comment to explain where it comes from may be helpful to readers.

Reply:
Added in 7979e32.